package.json

```diff
@@ -56,7 +60,15 @@
     "sonar-runner"
   ],
   "scripts": {
-    "test": "npx nyc mocha",
-    "ci": "npm run test && node ci-analysis.js"
+    "test-unit": "npx nyc mocha test/unit/**/*.test.js",
+    "test-integration": "npx nyc mocha test/integration/**/*.test.js",
+    "ci": "npm run test && node ci-analysis.js",
+    "format": "prettier --write \"{src,specs}/**/!(*.lint).js\""
```

Comment:
add assertions script

Reply:
build script which runs everything, executed in the ci. should fail if formatting fails

Reply:
do this outside of this PR